tox.ini

```diff
@@ -0,0 +1,45 @@
+[tox]
+requires =
+    tox>=4
+envlist =
```

Comment:
The lack of specified Python versions here will cause tox to run using the default system interpreter.